integration/test/crmapping/crmapping_test.go

```diff
@@ -22,15 +29,23 @@ import (
 	"github.com/giantswarm/azure-operator/v4/service/controller/resource/capzcrs"
 )
 
-func TestAzureConfigCRMapping(t *testing.T) {
+var update = flag.Bool("update", false, "update .golden reference files")
+
+// Test_AzureConfigCRMapping uses golden files.
+//
+//  go test ./integration/test/crmapping -run Test_AzureConfigCRMapping -update
+//
+func Test_AzureConfigCRMapping(t *testing.T) {
```

Comment:
These are actually unit tests at this point, but when getting this complete, my plan is to merge functionality of `azureconfig` and `capzcrs` resources to test cases such that they map each others after initial state and then the final state is verified.

Reply:
Bidirectional tests are now here. There's a lot of common functionality between these unit tests and bidirectional integration tests. What's your opinion @nprokopic & @axbarsan - should I move the unit tests out from here and move to `capzcrs` package or are they ok here?

Reply:
![](https://media.giphy.com/media/qDPg6HNz2NfAk/giphy.gif)

Reply:
No hard opinion for me, however you find it appropriate.

Once we remove all `AzureConfig` stuff, we will I guess remove also mapping tests?

Reply:
Yep, the mapping is gone then.

Reply:
![done](https://media.giphy.com/media/RkDX47fpp2nHlaZdjY/giphy.gif)